icons/postcss/postcss-plain-wordmark.svg

```diff
@@ -1 +1 @@
7-.3-1.336-.644-1.908-1.034-1.297-.83-1.946-1.894-1.946-3.191 0-1.116.41-1.945 1.227-2.49.817-.545 1.816-.817 2.996-.817s2.161.13 2.94.39l1.712 4.202h2.684l-.543-6.693a18.788 18.788 0 0 0-6.578-1.168zm19.734 0c-2.361 0-4.361.622-5.994 1.867a6.65 6.65 0 0 0-1.848 2.18c-.453.857-.681 1.914-.681 3.172 0 1.259.28 2.335.838 3.23a7.723 7.723 0 0 0 2.14 2.258 14.79 14.79 0 0 0 2.918 1.576c1.076.44 2.168.934 3.268 1.479 1.102.544 1.894 1.137 2.375 1.771.479.635.719 1.35.719 2.14 0 .792-.143 1.433-.428 1.926-.83 1.452-2.361 2.18-4.592 2.18a9.56 9.56 0 0 1-3.348-.623l-.7-4.164h-2.725l-.272 6.574c1.972.986 4.281 1.48 6.928 1.48 2.646 0 4.775-.597 6.383-1.79a6.781 6.781 0 0 0 2.043-2.393c.505-.973.76-2.21.76-3.717 0-1.504-.468-2.84-1.401-4.007-.494-.623-1.142-1.22-1.945-1.791a15.328 15.328 0 0 0-2.59-1.479 124.37 124.37 0 0 0-2.432-1.068c-.7-.3-1.336-.644-1.908-1.034-1.296-.83-1.945-1.894-1.945-3.191 0-1.116.409-1.945 1.226-2.49.818-.545 1.817-.817 2.998-.817 1.18 0 2.16.13 2.938.39l1.713 4.202h2.683l-.545-6.693a18.777 18.777 0 0 0-6.576-1.168zm-58.38.732-3.114.739v6.5h-3.463l-.273.388.545 2.53h3.19v11.832c0 1.53.382 2.634 1.15 3.308.765.674 1.705 1.012 2.822 1.012s2.024-.168 2.724-.506c.7-.338 1.232-.673 1.596-1.012-.493-1.53-.768-2.388-.818-2.57l-.002.002c-.285.233-.675.455-1.168.662-.234.13-.48.233-.739.31-.26.078-.578.118-.955.118a1.26 1.26 0 0 1-.931-.37c-.246-.246-.37-.785-.37-1.615V100.77h4.243v-2.684c0-.156-.066-.234-.196-.234h-4.046v-6.655l-.196-.584zM6.44 93.221H7.8c4.723 0 7.085 1.737 7.085 5.215 0 1.608-.508 2.893-1.52 3.853-1.141 1.09-2.905 1.635-5.29 1.635H6.438V93.22zm40.626 4.627c-1.66 0-3.088.441-4.28 1.324-1.273.96-1.907 2.232-1.907 3.814 0 1.608.584 2.905 1.752 3.89.57.468 1.277.91 2.121 1.325a64.24 64.24 0 0 0 3.035 1.38c1.18.507 1.77 1.239 1.77 2.198 0 .752-.298 1.324-.895 1.713a3.13 3.13 0 0 1-.855.352c-.312.078-.72.117-1.225.117-.506 0-1.058-.131-1.654-.39l-.934-2.257-4.01 1.518c.39 1.141 1.219 2.103 2.49 2.88 1.375.806 2.79 1.208 4.243 1.208 1.738 0 3.219-.419 4.437-1.248A5.214 5.214 0 0 0 52.672 114c.39-.674.584-1.538.584-2.59a4.625 4.625 0 0 0-.973-2.86c-.336-.467-.81-.907-1.42-1.321-.61-.416-1.484-.858-2.627-1.325-1.141-.467-2.036-.913-2.685-1.34-.65-.427-.973-.98-.973-1.654 0-.675.247-1.186.74-1.537a2.765 2.765 0 0 1 1.635-.525c.597 0 1.155.129 1.674.388l.894 1.985 3.97-1.518a6.95 6.95 0 0 0-2.296-2.607 7.232 7.232 0 0 0-4.127-1.244l-.002-.004zm-17.627.004c-2.335 0-4.253.804-5.76 2.412-.778.804-1.39 1.79-1.83 2.957-.441 1.167-.662 2.503-.662 4.01 0 2.957.78 5.304 2.336 7.042 1.53 1.764 3.504 2.647 5.916 2.647 2.308 0 4.229-.83 5.76-2.49 1.662-1.739 2.49-4.112 2.49-7.121 0-2.88-.777-5.177-2.334-6.89-1.556-1.711-3.53-2.567-5.916-2.567zm0 2.918c1.402 0 2.51.606 3.327 1.812.818 1.207 1.226 2.885 1.226 5.04 0 2.153-.429 3.755-1.285 4.804-.856 1.051-1.957 1.576-3.307 1.576-1.348 0-2.44-.537-3.27-1.615-.83-1.076-1.245-2.659-1.245-4.748 0-2.09.376-3.743 1.129-4.961.804-1.272 1.947-1.908 3.425-1.908z" style="stroke-width:.84605002"/></svg>
```

Comment:
Minimized using SVGOMG, and add fill color
```suggestion
<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 128 128">
  <path fill="#dd3a0a" d="M63.99 11.08a18.99 18.99 0 0 0-1.937.178c-.623.067-1.247.179-1.848.27-.733.088-1.468.132-2.203.222-.69.09-1.402.224-2.092.336-.245.045-.49.132-.736.176-.533.111-1.09.157-1.625.336-.956.29-1.913.627-2.848.982-.869.357-1.716.736-2.539 1.182-.846.467-1.648 1.022-2.45 1.558a38.321 38.321 0 0 0-2.25 1.627c-.71.559-1.378 1.183-2.025 1.807-.355.355-.646.78-.98 1.182-.556.647-1.112 1.27-1.646 1.939-.444.558-.847 1.138-1.27 1.719-.756 1.023-1.401 2.116-2.002 3.209-.245.446-.467.893-.69 1.338-.334.668-.69 1.314-1.001 2.005a8.477 8.477 0 0 0-.535 1.495 198.096 198.096 0 0 0-.913 3.298c-.267.982-.334 2.009-.355 3.012-.023.892.065 1.76.154 2.653.023.223-.11.49.156.667-.088.246-.022.382.11.424-.154.246.068.402-.02.625-.023.044.067.135.11.201 0 .068-.021.156 0 .223.023.2.113.402.113.625 0 .156.133.334.178.49.022.066 0 .178-.045.223-.134.156-.113.29.088.379.112.29.069.423.047.559.044.2-.023.265 0 .31.177.357.244.76.265 1.16 0 .09.045.178.09.29.022 0 0 .423 0 .423.11-.066.2-.088.31-.133.068.29.134.558.223.803.112.178.113.267.069.29.088.4.132.778.287 1.157 0-.423-.042-.846-.11-1.248.312.112.58.223.758.223.023-.068.044-.156.067-.201-.134-.133-.246-.333-.38-.467.045.178 0 .247 0 .314a3.974 3.974 0 0 0-.31-.136c.089-.11.177-.223.31-.356 0-.201-.108-.38-.421-.357.133-.044.266-.11.422-.156.245-.133-.022-.378-.022-.69.022-.378-.2-.758-.267-1.16-.112-.624-.09-1.56-.157-2.184-.133-.111-.243.179-.265.202.067.534.133 1.067.177 1.603-.132-.446-.223-.935-.334-1.47-.022.156-.044.222-.044.312-.11.11-.022.245.044.4-.155-.067-.334-.043-.334-.043v.623c-.221-.223-.287-.399-.4-.556-.044-.178-.067-.356-.11-.534.043-.112 0-.18 0-.246a13.445 13.445 0 0 1-.068-.537c.022-.022-.021-.355-.021-.355.178.02.29.043.4.043-.222-.288-.067-.466.024-.645.045-.09.199-.11.244-.201.177-.334.469-.6.469-1.025 0-.2.066-.424.087-.625.045-.38.09-.758.112-1.137.045-.49-.001-.981.111-1.45a31.8 31.8 0 0 1 .647-2.341c.177-.558.38-1.09.601-1.649.334-.87.645-1.74 1.069-2.564.6-1.205 1.29-2.364 1.959-3.545.423-.736.869-1.427 1.337-2.14.445-.692.89-1.404 1.403-2.051a18.135 18.135 0 0 1 2.36-2.52c.89-.802 1.736-1.627 2.692-2.363a27.068 27.068 0 0 1 3.963-2.543c1.137-.6 2.316-1.072 3.496-1.518.758-.289 1.58-.422 2.36-.668a9.37 9.37 0 0 1 2.717-.445c.6-.023 1.178-.09 1.779-.156a46.24 46.24 0 0 0 1.648-.156c-.067.088-.089.133-.111.177-.244.357-.51.714-.756 1.07-.2.29-.422.604-.623.893-.357.47-.737.937-1.07 1.428-.512.736-.98 1.493-1.514 2.228-.533.714-1.113 1.384-1.67 2.075-.734.96-1.426 1.94-2.16 2.898-.668.914-1.379 1.786-2.025 2.7-.579.8-1.09 1.649-1.647 2.472-.823 1.227-1.647 2.43-2.47 3.656-.647.936-1.292 1.85-1.96 2.787-.044.068-.111.088-.177.133-.178.29-.179.447-.246.58-.178.334-.356.668-.557.98-.112.178-.267.334-.4.512-.022-.267.355-.4.177-.736-.11.18-.223.335-.334.514l-.267.533c.087.067.238.112.094.24-.015-.052-.09-.088-.147-.13-.178.245-.355.512-.533.802.134.11.2.069.289-.088.078-.156.19-.278.291-.41-.035.065-.07.13-.106.19a.8.8 0 0 0 .245-.223c0 .245-.133.444-.245.644-.088.178-.31.313-.466.446.022.022.176.09.199.113.2-.068.268-.18.29-.113.069-.09.135-.156.157-.223.088-.223.221-.4.444-.49.09-.045.155-.112.222-.178.669-.692 1.359-1.36 1.96-2.096.935-1.138 1.781-2.32 2.673-3.478.377-.49.822-.98 1.223-1.45.066.513.022 1.004 0 1.473-.068.824-.112 1.65-.178 2.496-.023.335.001.667-.022 1.002 0 .09-.068.157-.068.246 0 .223.068.468.068.692-.067.825-.157 1.626-.224 2.451-.023.178.065.357.11.514-.11 1.203.136 2.475-.065 3.746.155-.246.199-.401.199-.557.021-.134 0-.268 0-.424v-.222c.021-.402.11-.825.066-1.227-.044-.446-.044-.869.045-1.293.045-.178-.022-.402.09-.625v.334c-.022 1.27.024 2.542-.154 3.813-.023.111.067.311-.178.289-.201.4.044.691-.178.914-.045.045.042.245.065.379-.09.268-.089.269.045.29.022.491.066.936.088 1.405.355-.846.178-1.718.336-2.588.022.357.044.714.021 1.092 0 .581-.023 1.137-.045 1.717 0 .111-.023.224-.045.336-.067.313-.087.624-.199.914-.089.223-.045.557-.045.824v.847c.089.38-.022.58.045.803-.068.224-.09.29-.068.358l.068.334c.446-.045.244.29.422.468-.134.18-.088.246-.088.336 0 .2-.022.38-.045.58 0 .513-.021 1.025-.021 1.516 0 .068.021.132.021.2 0 .045-.021.626-.021.626.044.066-.001.133-.069.2-.02 0-.088-2.833-.088-2.833h-.334c-.044 1.337.068 2.542-.177 3.791-.268.022-.356.067-.647.067-.69 0-1.38 0-2.07.021-.824 0-1.625.022-2.45.022-.399 0-.823.045-1.224.045-.535 0-1.09-.067-1.623-.067-.825 0-1.627.068-2.451.09-.6.022-1.2.022-1.801.045-.402.021-.804.042-1.203.088-.178.022-.334.087-.557.154-.067 0-.222-.022-.355 0-.224.045-.424.18-.649.225-.334.068-.423.155-.379.49v.223c-.022.044.49-.069.49-.069-.045 0-.066.135-.134.246.18-.067.29-.112.402-.156.334.157.624.07.89.137.869.222 1.757.311 2.647.398.58.045 1.136.07 1.715.09.823.045 1.648.09 2.47.09.691 0 1.361-.068 2.05-.068.956 0 1.914.044 2.87.068.201 0 .425-.023.647-.023h1.623c1.449-.044 2.896-.07 4.344-.112.825-.023 1.671-.067 2.494-.09l2.537-.068c.98-.022 1.96-.02 2.918-.04.735 0 1.448-.027 2.184-.048.822-.022 1.668-.089 2.492-.133.446-.022.891-.067 1.336-.11.535-.045 1.092-.11 1.625-.134.802-.045 1.626-.068 2.428-.113 1.001-.066 2.003-.155 3.027-.178.936-.044 1.893-.02 2.828-.043.824-.023 1.648-.068 2.494-.09l.936-.044 1.65 9.15c-.098.07-.193.14-.293.21-.733.514-1.468 1.026-2.203 1.516-.4.267-.846.468-1.27.713-.422.246-.824.469-1.247.713-.268.178-.512.423-.801.559-.735.313-1.515.579-2.272.869-.736.313-1.27.58-1.826.781a12.56 12.56 0 0 1-1.58.422c-.178.046-.38 0-.58.024-.379.067-.756.157-1.135.2-.98.134-1.96.224-2.94.358-.579.068-1.135.177-1.714.244-.4.022-.824-.021-1.225-.021-.289 0-.6.068-.89.023-.467-.067-.935-.178-1.403-.267-.11-.022-.267-.068-.355-.022-.178.09-.335-.044-.514 0-.11.023-.22-.068-.332-.068-.09-.021-.203.023-.314 0-.246-.066-.468-.155-.737-.133-.264.022-.532-.112-.82-.178-.402-.09-.779-.177-1.182-.267a2.013 2.013 0 0 1-.443-.201c-.58 0-.958-.022-1.314-.157-.312-.11-.602-.267-.936-.222-.088.022-.153-.022-.244-.022-.068-.09-.133-.267-.2-.267-.444.022-.712-.357-1.091-.47-.2-.067-.356-.288-.557-.355-.49-.2-.888-.624-1.445-.691-.178-.023-.355-.112-.557-.178.068.088.11.156.178.223-.645-.313-1.246-.67-1.848-1.004-.088-.022-.155-.11-.222-.156l-.6-.4c-.334.067-.468-.068-.558-.112-.601-.423-1.292-.692-1.871-1.16-.246-.202-.534-.335-.711-.58-.313-.38-.69-.647-1.114-.87-.333-.177-.645-.379-.957-.58-.156-.111-.312-.244-.312-.49 0-.156-.065-.312-.266-.222-.089-.134-.177-.29-.289-.334-.267-.11-.446-.315-.646-.493-.2-.177-.355-.422-.6-.511-.446-.178-.714-.555-.98-.867-.625-.737-1.182-1.54-1.76-2.319-.045-.068-.135-.113-.201-.18.088.246.201.446.312.668-.222-.088-.378-.244-.512-.4-.022.023 1.535 2.586 1.535 2.586.136.2.246.401.38.602a.687.687 0 0 1-.669-.358c-.2-.355-.422-.577-.867-.531a3.91 3.91 0 0 1-.111-.492c-.09 0-.156.001-.313-.022.313.49.58.914.87 1.405-.113-.068-.179-.113-.225-.135.4.423.69.958 1.201 1.226.112.512.69.758.824 1.248-.11-.022-.2-.043-.312-.066-.178-.223-.379-.446-.578-.691.044.155.11.29.199.402.267.334.513.667.824.957.112.113.268.159.268.36.177.088.244.109.31.109.046.135-.02.336.246.29.178.335.357.336.602.225.29.29.447.313.646.135 0 .11.022.22.022.31.089-.044.068-.11.068-.222.11.089.177.134.221.178.222.313.423.602.734.847.423.357.802.736 1.202 1.07.78.648 1.65 1.183 2.517 1.673 1.07.602 2.093 1.249 3.184 1.828 1.313.692 2.717 1.162 4.164 1.496.534.111 1.046.22 1.582.353.465.136.932.337 1.4.448.713.156 1.448.29 2.16.423.155.023.313.045.49.067.2.045.444-.044.489.023.112.178.2.089.334.067.155-.045.334-.045.512-.045.623.022 1.224.155 1.847.111.136 0 .336-.113.447.11.2-.09.312-.089.4-.11.025.2 0 .2-.042.223-.046.022-.112.044-.201.111.8.156 1.558.178 2.224 0 1.16.089 2.162-.045 3.12-.379.088.112.155.268.289.045.044-.068.156-.091.246-.113.2-.045.445-.135.623-.069.201.068.267-.109.422-.109.268 0 .512-.024.757-.068.134-.022.269-.088.424-.133a1.078 1.078 0 0 0-.49-.113c.268-.134.512-.246.758-.356-.045.066-.09.156-.178.156h.31c-.067 0-.156.2-.267.334.8-.2 1.535-.379 2.248-.558-.245-.467.267-.31.445-.578a6.931 6.931 0 0 1-.67.177c.758-.378 1.537-.712 2.293-1.047-.11.134-.244.2-.443.313.134.021.222.022.29.043.354.112.267-.223.378-.424.423 0 .6-.065.8-.133.223-.067.445-.134.669-.222.355-.156.69-.356 1.047-.534.067-.023.176 0 .265 0 .246-.157.334-.291.4-.27.314.046.534-.067.624-.355l.064-.046c.09.197.267.219.557.02-.179 0-.356-.022-.512-.044-.016 0-.022.01-.031.016.11-.082.213-.161.3-.215.49-.267.955-.534 1.446-.8.178 0 .268-.001.312-.024a9.67 9.67 0 0 0 .825-.446 1.865 1.865 0 0 1-.223-.111c.162-.033.227-.056.31-.082l.022.12.002.001-.022-.121c.03-.01.076-.018.112-.03.533-.625 1.268-1.05 1.824-1.54.245-.022.246-.09.246-.133.088-.112.177-.244.287-.334.49-.446 1.025-.825 1.47-1.293a31.676 31.676 0 0 0 1.849-2.008c.533-.624 1.068-1.248 1.513-1.94.624-.958 1.158-1.94 1.715-2.945.401-.712.756-1.425 1.158-2.138.043-.068.133-.157.2-.157.6-.067 1.204-.11 1.783-.177.133-.022.288-.11.422-.178-.068-.112-.112-.246-.178-.357-.09-.136-.223-.244-.313-.38-.088-.11-.155-.245-.222-.378-.068-.11-.133-.2-.2-.313a8.123 8.123 0 0 1-.38-.758c-.022-.067-.069-.176-.045-.244.4-.869.557-1.806.935-2.676.378-.891.49-1.894.668-2.853.156-.914.246-1.829.356-2.742.067-.647.156-1.27.177-1.916.136-1.026.09-1.986.069-2.946-.022-.846-.07-1.714-.112-2.562l-.001-.004c-.044-.87-.065-1.738-.155-2.607-.155-1.182-.313-2.386-.58-3.545-.2-.959-.532-1.874-.867-2.787-.423-1.07-.846-2.163-1.38-3.188-.869-1.65-2.005-3.122-3.052-4.66-.692-.981-1.581-1.784-2.25-2.787-.378-.58-.935-1.069-1.492-1.514-.89-.715-1.825-1.363-2.76-1.986a27.516 27.516 0 0 0-2.183-1.383c-.6-.336-1.246-.602-1.87-.893-.712-.334-1.403-.736-2.204-.892-.202-.042-.378-.11-.579-.178-.511-.178-1.003-.377-1.515-.533-.779-.245-1.536-.535-2.338-.736a21.44 21.44 0 0 0-2.379-.448c-.89-.13-1.804-.176-2.695-.263a22.674 22.674 0 0 0-1.713-.157c-.846-.044-1.715-.044-2.56-.066-.424-.023-.869-.068-1.292-.045zm1.906 2.584c.29.046.56.089.827.111l1.068.069c.801.065 1.602.109 2.402.219.669.09 1.337.225 2.028.335 1.47.246 2.917.648 4.32 1.116.824.267 1.624.624 2.428.959 1.802.736 3.517 1.626 5.209 2.562.533.29.846.758 1.246 1.182.58.602 1.159 1.204 1.715 1.828.644.736 1.244 1.494 1.869 2.252.556.668 1.135 1.338 1.693 2.008.178.2.313.445.467.668.355.535.757 1.048 1.068 1.627.222.424.356.916.512 1.384.222.692.445 1.36.668 2.051.155.513.287 1.05.398 1.582.09.358.068.735.178 1.09.136.47.002.982.137 1.451.021.045.04.112.064.156.09.088.157.18.024.313-.021.022-.021.11 0 .178.045-.044.066-.09.111-.157 0 .045.022.07.022.09 0 .088-.023.177 0 .266 0 .044.044.11.066.156.045-.046.089-.069.178-.137.2.76.222 1.517.222 2.297 0 .736.133 1.473-.177 2.186.066-.245-.09-.469-.024-.692l.01.006c-.023 0-.156-.136-.178-.158-.023.068 0 .135-.045.225-.022-.066-.177-.046-.177-.112-.023.312.134.626.11.938.069-.045.334-.069.401-.135a.177.177 0 0 0 .09.045c-.446.335-.335.668-.135 1.049.044.088.179.267.045.445-.044.045.023.177.045.29h.065v.042l-.2.069c-.044-.068-.31-.202-.31-.315v1.139c.222.067.222.38.31.537a.663.663 0 0 0 .09-.178l.024.02c-.023.133-.045.268-.09.402-.044.2-.088.378-.11.578 0 .022.042.067.042.088-.244.558-.109 1.181-.198 1.76-.067.423 0 .873-.045 1.318-.045.38-.136.78-.224 1.158-.023.068-.068.088-.178.133a1.789 1.789 0 0 0-.021-.69c-.024 0-.314-.156-.336-.156 0 .314.176.737.244 1.092-.068-.022-.335-.177-.356-.177-.045.267-.044.622-.066.912-.023.223-.223.424-.246.67 0 .135.157.222.135.445h.177c0-.445.201-.846.201-1.315h.065c.023.045.09.09.068.112-.045.245-.135.514-.111.76 0 .133.11.266.068.421a2.165 2.165 0 0 0-.091.47c-.023-.113-.045-.157-.045-.223-.022 0-.222-.024-.245-.024-.022.134.156.27.133.38-.022.023-.043.023-.088.046l-.734-1.072c-.58-.846-1.16-1.694-1.738-2.563a5.36 5.36 0 0 1-.403-.67 31.246 31.246 0 0 0-2.293-3.412c-.644-.824-1.159-1.717-1.76-2.584-.554-.825-1.11-1.627-1.667-2.453-.2-.267-.445-.513-.623-.803-.578-.914-1.093-1.85-1.67-2.763-.491-.76-1.023-1.496-1.535-2.23-.178-.246-.356-.515-.536-.782a.311.311 0 0 1-.043-.154c-.044-.491-.09-.982-.113-1.45 0-.446.045-.89.045-1.36v-.4c0-.111.022-.201.022-.314.02-.021.044-.066.066-.088-.646 0-1.314-.023-1.959 0-.423.021-.646-.156-.936-.469-.846-.958-1.468-2.05-2.203-3.076-.69-.981-1.403-1.942-2.115-2.9-.58-.78-1.16-1.538-1.738-2.318-.647-.892-1.27-1.783-1.938-2.674a323.274 323.274 0 0 0-2.449-3.211c-.38-.49-.803-.957-1.227-1.47zm-2.064 1.17c.048.008.092.048.137.104.6.758 1.202 1.538 1.802 2.296.447.559.892 1.092 1.315 1.672.512.714 1 1.428 1.514 2.14.578.826 1.158 1.649 1.714 2.495.847 1.25 1.649 2.5 2.495 3.748.623.914 1.247 1.805 1.847 2.719H73.52c-1.046-.021-2.117-.11-3.163-.067-1.424.068-2.87.225-4.294.336-1.804.157-3.63.178-5.432.178-.647 0-1.293.09-1.938.135-.78.044-1.537.09-2.316.111-.914.046-1.803.067-2.717.09-.513.022-1.047 0-1.603 0l.002-.004c.71-.981 1.445-1.964 2.158-2.967.222-.292.4-.603.623-.892.068-.09.157-.133.267-.223l-.023-.043c.089-.112.178-.203.223-.29.113-.179.18-.38.27-.581a.314.314 0 0 1 .089-.086c.313.156.312-.178.422-.29.2-.224.4-.469.578-.692.134-.178.27-.335.38-.535.023-.068-.045-.178-.067-.268h-.067c.068-.044.134-.11.201-.156.022 0 .021 0 .043-.022.134-.201.269-.401.403-.58.266-.355.555-.712.867-1.023a4.15 4.15 0 0 1-.135.31c.023 0 .023.024.045.024.267-.357.557-.691.824-1.05-.023-.02-.045-.044-.068-.044-.11.112-.2.224-.31.336-.023-.023-.045-.045-.07-.045.225-.29.45-.58.67-.914.046.068.089.155.177.334.09-.223.18-.379.246-.535-.066.067-.154.133-.221.2l-.068-.066.4-.601c.423-.602.846-1.226 1.27-1.828.156-.223.354-.402.49-.625.423-.846 1.092-1.517 1.603-2.297.112-.156.267-.288.4-.422.056-.068.106-.09.153-.082zm-6.277 8.684c-.133.088-.246.156-.38.246h-.001c-.023.022-.044.023-.022.023.023.11.021.199.043.354.157-.225.267-.4.403-.578l-.043-.045zM76.344 32.7c.2 0 .312.069.424.246.2.334.4.67.623 1.004.022.044.066.11.066.178 0 .49 0 1.003.022 1.494.02.513.069 1.003.09 1.516 0 .512-.047 1.047-.024 1.582.023.759.045 1.54.045 2.318l.004.002v.201c-.022 0-.045.024-.09.024-.089-.313-.177-.65-.29-.961-.222-.648-.467-1.295-.868-1.873-.447-.624-.845-1.27-1.291-1.916-.11-.157-.269-.291-.403-.424-.468-.514-.934-1.05-1.535-1.428a18.187 18.187 0 0 0-2.785-1.428 6.22 6.22 0 0 1-.777-.4h.533l3.807-.068c.824-.022 1.626-.022 2.449-.067zm-16.719.582c.161-.002.322.002.488.024-.355.179-.712.357-1.023.58-.401.267-.778.624-1.18.914a6.73 6.73 0 0 0-1.092 1.025c-.423.512-.89.959-1.336 1.45-.313.356-.669.69-.935 1.07-.378.49-.712 1.023-1.068 1.582.111-2.119.245-4.213.156-6.354h.002c1.046-.045 2.047-.111 3.072-.156.8-.045 1.625-.067 2.426-.111.167-.01.329-.021.49-.024zm5.586 1.139c.666.021 1.334.021 2.002.066a3.4 3.4 0 0 1 .824.178 8.4 8.4 0 0 1 1 .424c.378.203.735.424 1.092.67.645.423 1.247.89 1.892 1.27.624.378 1.113.892 1.514 1.472.49.69 1.047 1.315 1.27 2.162l.334 1.473c.067.267.223.534.156.845-.113.559-.202 1.138-.291 1.696-.044.224-.09.448-.067.668.043.693-.223 1.293-.49 1.918-.09.2-.223.402-.334.625.022.022.156.09.178.09l.201-.4c.023 0 .023-.002.045.02-.112.312-.202.647-.314.96-.266.69-.533 1.405-.868 2.074-.177.379-.489.69-.71 1.049-.222.311-.401.645-.624.935-.2.246-.4.49-.667.668-.601.446-1.226.891-1.872 1.248-.4.246-.867.38-1.312.492l.006-.004c-.713.178-1.448.38-2.184.47-.623.089-1.268.068-1.914.068-.58 0-1.136-.023-1.715-.069-.645-.066-1.292-.133-1.916-.424-.535-.245-1.09-.468-1.623-.736-.536-.267-1.07-.578-1.472-1.068-.288-.358-.69-.647-1.045-.96.067.268.221.492.466.647-.111.134-.266.067-.49-.29-.066.044-.133.067-.178.089.068-.156.135-.312.202-.445-.09-.178-.203-.38-.291-.559-.178-.29-.356-.6-.512-.89-.045-.068-.044-.135-.088-.202a4.982 4.982 0 0 1-.312-.402c-.178-.267-.335-.556-.557-.824 0 .067.023.135.023.224-.023 0-.045.022-.045.022l-.468-1.137c-.022 0-.023 0-.045.022l.4 1.472c-.022 0-.045.022-.068.022a1.37 1.37 0 0 1-.178-.29c-.044-.179-.177-.177-.287-.11-.068.044-.113.2-.09.267.157.379.335.758.514 1.137.044.133.087.245.154.379.023.044.09.089.09.156.09.178.157.354.267.512.201.267.423.535.623.802.2.268.379.558.557.825.022.044.088.045.11.068.112.334.49.579.757.445.068.513.624.691.846 1.182a10.009 10.009 0 0 1-.645-.291c-.022.022-.024.045-.046.09.267.133.535.266.779.422.224.135.446.269.646.423.268.246.58.381.957.47a.24.24 0 0 0-.043-.093c.623.29 1.27.604 1.758.848-2.315.046-4.786.09-7.347.135 0-.423-.022-.826-.022-1.25 0-.178.044-.333.022-.512-.068-1.024-.069-2.051-.202-3.055-.132-1.07-.132-2.117-.109-3.19.021-.577-.024-1.178-.047-1.759 0-.78 0-1.54.024-2.318 0-.112.045-.2.09-.313h.068c.021.046 0 .112.043.135.067.068.178.178.222.156.133-.09.335-.177.38-.312.11-.267.156-.557.156-.824 0-.2.09-.335.177-.47.046-.067.067-.133.112-.222-.09-.068-.178-.11-.29-.2.09-.244.069-.536.403-.669.044-.023.045-.135.068-.201.021-.18.041-.381.065-.559.044.068.069.09.111.133.024-.044.045-.066.045-.066 0-.088-.022-.268 0-.268.267-.045.155-.333.29-.467.02-.023-.022-.092-.022-.136 0-.044 0-.11.021-.11.31-.068.266-.402.467-.558.068-.045.089-.11.178-.223.177.311-.135.469-.157.691.378-.222.49-.757.268-.914.089-.067.176-.156.244-.246.245-.267.511-.512.734-.78.157-.199.247-.445.403-.646.09-.111.267-.2.4-.267.401-.222.778-.467 1.024-.846.044-.045.111-.09.177-.135.068-.044.201-.044.223-.111.023-.044-.043-.156-.088-.223.088-.044.178-.133.356-.222-.068.133-.09.223-.135.312l.111.112c.202-.156.513-.334.713-.467 0-.021-.111-.112-.111-.112-.133.068-.334.178-.533.268.042-.068.04-.11.066-.133.31-.2.623-.401.957-.558.778-.356 1.559-.713 2.338-1.047.134-.068.312-.022.467-.022a.5.5 0 0 0 .177-.023c.356-.112.713-.223 1.069-.313.155-.044.334-.066.492-.066zm-.867.332c-.245.112-.444.2-.645.289h-.002c0 .023-.11.201-.09.201.112-.022.359-.2.493-.223.066 0 .155.023.222.045 0-.09 0-.178.022-.312zm16.097 3.795c.223.29.468.56.623.847.513.938 1.18 1.74 1.803 2.586.535.736 1.025 1.54 1.516 2.297.6.914 1.18 1.827 1.781 2.74a68 68 0 0 0 1.379 2.053c.423.601.87 1.182 1.314 1.762.378.513.779 1.026 1.157 1.539.446.624.89 1.247 1.336 1.85.221.311.467.623.71.935.136.178.336.314.47.424a.173.173 0 0 1-.089-.023c.068.088.111.156.157.2.243.133.264.266.134.464-.022.044-.066.09-.087.136-.023-.045-.023-.112-.045-.203-.134.358-.425.18-.625.18-1.202.023-2.426.021-3.63.021-1.157 0-2.336.002-3.493.022-.38 0-.78.088-1.159.111-.577.028-1.155.039-1.732.055l-1.293-7.166-.002.004 1.291 7.164c-.334.01-.668.023-1.002.031-.068 0-.134-.11-.246-.2a1.498 1.498 0 0 1-.133.202c-.024-.044-.068-.087-.113-.154a1.277 1.277 0 0 1-.067.176h-.064c-.045-.312-.135-.6-.135-.913 0-.668.069-1.337.09-2.007.023-.312 0-.6 0-.891.09.068.155.113.244.178.044-.087.09-.155.135-.221 0 0-.045-.024-.09-.068.042-.023.088-.068.088-.09.022-.068.068-.134.068-.178 0-.156-.046-.289-.068-.422.022 0 .045 0 .068-.023l.133.668h.067c-.09-.155.2-.29-.045-.467-.022-.023.023-.158.023-.225-.068-.132-.11-.264-.178-.443a.531.531 0 0 1 .155-.068c-.09-.044-.156-.088-.223-.11.223-.201.224-.314.023-.537.046-.021.088-.02.088-.04 0-.202.001-.404-.043-.602-.044-.225.068-.49-.267-.625a.651.651 0 0 0 .09.177s-.07.022-.135.067c-.29-.268-.11-.625-.178-1.026.156.045.268.09.38.135-.046-.157-.157-.337-.224-.537h-.09v-1.47c0 .022.155.021.245.044 0-.023.021-.045.021-.068-.067-.089-.157-.178-.246-.29l.045-.044c-.178-.222-.088-.532-.088-.6v-1.56c.224.223-.11.513.223.646v-.222h-.024c-.022-.335-.02-.67-.043-1.004h-.045c-.022.045-.022.068-.044.113h-.043l.132-5.33zm.225 10.834v-.004c-.066-.022-.154-.065-.31-.11.044.089.067.154.09.221.11-.022.155-.02.199-.043 0-.021.002-.043.021-.064zm-37.738-9.01c-.266.312-.51.625-.756.936h-.004c.044.022.11.177.156.222.245-.334.58-.755.824-1.068-.022-.021-.2-.068-.22-.09zm-1.492 1.094c-.134.2-.466.557-.6.758.022.022.267-.09.289-.067.133-.2.267-.424.4-.625a.355.355 0 0 1-.09-.066zm.466.332c-.132.091-.266.204-.4.293.045.044.045.067.045.111-.178-.023-.245.069-.334.313-.022.067-.133.087-.244.175-.022.046-.045.16-.09.27l.023.023c-.044.045-.111.069-.111.069-.134.133-.29.244-.424.377-.022.067-.045.134-.045.2.157-.155.425-.446.625-.624l.022.023c.089-.133.156-.266.246-.4.021.022.022.043.045.066-.068.178-.135.378-.201.557-.047.022-.114.068-.114.045-.089.068-.156.156-.246.223l-.4.267c-.223.2-.444.377-.69.578-.155.246-.31.468-.49.758.268-.068.335.046.313.27.311.044.333-.2.422-.38.022-.045.045-.136.09-.158.2-.177.421-.333.644-.511.045-.045.045-.135.045-.201 0-.022-.111-.023-.178-.045.2-.224.49-.6.69-.846.111-.068.222-.158.222-.225-.018-.2.092-.276.227-.347-.106.477-.397.884-.692.968.09.045.179.089.246.067.066-.023.11-.135.155-.201l.6-.87.202-.335-.09-.088v.002c-.088.155-.178.288-.267.445l-.074-.037.006-.004c-.09-.068-.112-.134-.112-.156v-.002a.927.927 0 0 1 .114-.201c.044 0 .109-.022.109-.022.09-.156.09-.269.18-.402-.022-.022-.046-.022-.069-.045zm38.633 2.738-.002.004c-.022.066-.021.132-.021.2 0-.068 0-.136.023-.204zm-.023.426v.023c.023.022.023.022.045 0-.022 0-.023 0-.045-.023zm0 .045c-.065.022-.131.022-.221.045.11.041.217.064.22.107v-.152zm0 .152v.004l.002-.002-.002-.002zm-3.276 3.908c-.045 2.542.245 5.038.578 7.557l-7.656.268c1.001-.423 1.825-1.025 2.58-1.717a2.55 2.55 0 0 1 .358-.313c.935-.578 1.623-1.381 2.248-2.273.712-1.026 1.18-2.185 1.76-3.278.044-.09.087-.176.132-.244zm3.162.42.002.002c.022 0 .043.024.043.024l-.021-.026c0-.02-.002-.02-.024 0zm-45.418.07a.557.557 0 0 0-.199.09c.09.223.2.468.29.692a.694.694 0 0 1 .177-.088c-.09-.223-.177-.469-.266-.693h-.002zm-.623.917c-.066.111-.134.222-.11.31 0 .067.11.132.177.2 0-.022.022-.044.045-.065-.044-.157-.066-.29-.112-.445zm1.157.511a.768.768 0 0 0-.2.09c.178.378.356.78.534 1.158.022 0 .2-.09.2-.09-.179-.38-.289-.802-.534-1.158zm-.444.891c0 .088-.113.065-.113.154-.131 0-.242-.021-.353-.021-.022.022.067.157.09.18h.466c0-.113-.023-.201-.023-.313h-.067zm20.479.87c.245.378.491.736.758 1.09l-.022.024c-.046-.022-.112-.045-.156-.09l-.535-.802c-.044-.066-.069-.134-.045-.223zM80.4 53.948l-.002.002c-.022.268-.044.468-.044.668 0 .045.045.133.068.133.044.024.133-.022.178-.043a.17.17 0 0 0 .045-.09c-.067-.046-.134-.067-.202-.113.423-.223.001-.334-.043-.557zm10.159 3.903c.31 0 .644.044 1.045.066-.178.156-.289.246-.422.357.088.022.178.068.267.112a.604.604 0 0 1-.355.111c-.046.113-.092.158-.137.38h.402c-.155 0-.29.201-.445.335-.178-.046-.335-.001-.424.244.156.088.134.156.112.246-.268.088-.468.178-.356.467-.09.044-.2.09-.312.156-.11.178-.312.222-.268.467.09.021.178.068.246.088-.112.045-.222.112-.334.156-.4.245-.222.357-.133.469a.938.938 0 0 1-.2.111c.043.2.11.246.132.246-.062.046-.149.118-.232.186a.477.477 0 0 1 .027-.072h-.09c0 .04.01.075.016.11-.02.018-.033.026-.055.044-.068.2-.157.288-.246.422-.089.133-.133.314-.244.447-.467.514-.802 1.115-1.403 1.516-.223.156-.422.357-.623.558-.4.378-.756.803-1.18 1.16-.463.39-.945.748-1.44 1.102l-1.651-9.148.467-.024s.934-.023 1.402-.068c.801-.044 1.602-.133 2.404-.155.511-.022 1.024 0 1.535-.023.824-.022 1.67-.066 2.495-.066zm-48.942 8.716c.243.269.488.536.733.827.067-.023.135-.023.2-.045.068.111.113.245.179.379-.312-.023-.468-.157-.38-.334-.133-.068-.289-.112-.355-.202-.156-.177-.29-.379-.445-.556l.068-.069zm43.348 4.194c-.18.042-.314.067-.424.111-.046.09-.09.155-.135.244.023 0 .046.023.069.045.133-.134.288-.245.49-.4zm-1.139.623c-.245.156-.512.31-.758.467.022.022.024.046.047.068.312-.067.558-.223.803-.4l-.002-.002c-.022-.045-.067-.09-.09-.133zM72.92 75.623c-1.024.29-1.357.333-1.469.178.512-.068.98-.112 1.469-.178zm-1.516.2c.023.043.023.088.045.132-.133 0-.222.047-.334.068v-.002c-.31.068-.645.114-.957.159-.111.022-.245.133-.289-.069-.045.023-.089.024-.133.047.11.334.377.066.6.2-.49.067-.955.111-1.424.177v-.045c.135-.022.29-.043.422-.066v-.067a6.512 6.512 0 0 0-.668-.044c-.065 0-.154.089-.22.11-.09.023-.18.046-.292.046-.378.022-.756.023-1.113.045-.023 0-.068-.024-.09-.024v-.066c1.492-.2 2.962-.401 4.453-.602zm-4.652.577v.067a12.25 12.25 0 0 1-1.246.133v-.065c.423-.044.823-.09 1.246-.135zM.096 89.873v2.959l2.45.193v20.471l-2.49.545.39 2.725h11.519l.117-2.608c-3.113-.389-4.994-.648-5.643-.777v-6.111h1.79c3.295 0 5.851-.662 7.667-1.985 2.051-1.506 3.075-3.775 3.075-6.81 0-2.596-.832-4.645-2.492-6.15-1.842-1.635-4.566-2.452-8.172-2.452H.096zm81.806.008c-3.71 0-6.772 1.155-9.185 3.465-2.647 2.54-3.969 5.967-3.969 10.273 0 4.073 1.39 7.368 4.164 9.885 2.543 2.283 5.603 3.426 9.184 3.426 2.958 0 5.254-.585 6.888-1.752l-1.129-3.074c-.416.31-1.089.634-2.023.972-.934.337-1.957.506-3.074.506-3.009 0-5.422-.986-7.238-2.957-1.79-1.946-2.688-4.41-2.688-7.395 0-2.75.82-5.085 2.455-7.005a8.693 8.693 0 0 1 2.86-2.18 8.005 8.005 0 0 1 3.521-.816c1.22 0 2.075.09 2.568.271l1.518 4.203H88.4l-.738-7.082c-2.18-.494-4.099-.74-5.76-.74zm18.524 0c-2.362 0-4.36.622-5.994 1.867a6.667 6.667 0 0 0-1.85 2.18c-.453.857-.682 1.914-.682 3.172 0 1.259.28 2.335.838 3.23a7.723 7.723 0 0 0 2.14 2.258 14.87 14.87 0 0 0 2.92 1.576c1.078.44 2.166.934 3.27 1.479 1.103.544 1.895 1.137 2.375 1.771.48.635.72 1.35.72 2.14 0 .792-.144 1.433-.429 1.926-.83 1.452-2.36 2.18-4.591 2.18a9.56 9.56 0 0 1-3.348-.623l-.7-4.164h-2.724l-.273 6.574c1.972.986 4.281 1.48 6.927 1.48 2.647 0 4.774-.597 6.381-1.79a6.787 6.787 0 0 0 2.045-2.393c.505-.973.758-2.21.758-3.717 0-1.504-.467-2.84-1.4-4.007-.495-.623-1.142-1.22-1.946-1.791a15.328 15.328 0 0 0-2.59-1.479c-.921-.415-1.73-.77-2.431-1.068-.7-.3-1.336-.644-1.908-1.034-1.297-.83-1.946-1.894-1.946-3.191 0-1.116.41-1.945 1.227-2.49.817-.545 1.816-.817 2.996-.817s2.161.13 2.94.39l1.712 4.202h2.684l-.543-6.693a18.788 18.788 0 0 0-6.578-1.168zm19.734 0c-2.361 0-4.361.622-5.994 1.867a6.65 6.65 0 0 0-1.848 2.18c-.453.857-.681 1.914-.681 3.172 0 1.259.28 2.335.838 3.23a7.723 7.723 0 0 0 2.14 2.258 14.79 14.79 0 0 0 2.918 1.576c1.076.44 2.168.934 3.268 1.479 1.102.544 1.894 1.137 2.375 1.771.479.635.719 1.35.719 2.14 0 .792-.143 1.433-.428 1.926-.83 1.452-2.361 2.18-4.592 2.18a9.56 9.56 0 0 1-3.348-.623l-.7-4.164h-2.725l-.272 6.574c1.972.986 4.281 1.48 6.928 1.48 2.646 0 4.775-.597 6.383-1.79a6.781 6.781 0 0 0 2.043-2.393c.505-.973.76-2.21.76-3.717 0-1.504-.468-2.84-1.401-4.007-.494-.623-1.142-1.22-1.945-1.791a15.328 15.328 0 0 0-2.59-1.479 124.37 124.37 0 0 0-2.432-1.068c-.7-.3-1.336-.644-1.908-1.034-1.296-.83-1.945-1.894-1.945-3.191 0-1.116.409-1.945 1.226-2.49.818-.545 1.817-.817 2.998-.817 1.18 0 2.16.13 2.938.39l1.713 4.202h2.683l-.545-6.693a18.777 18.777 0 0 0-6.576-1.168zm-58.38.732-3.114.739v6.5h-3.463l-.273.388.545 2.53h3.19v11.832c0 1.53.382 2.634 1.15 3.308.765.674 1.705 1.012 2.822 1.012s2.024-.168 2.724-.506c.7-.338 1.232-.673 1.596-1.012-.493-1.53-.768-2.388-.818-2.57l-.002.002c-.285.233-.675.455-1.168.662-.234.13-.48.233-.739.31-.26.078-.578.118-.955.118a1.26 1.26 0 0 1-.931-.37c-.246-.246-.37-.785-.37-1.615V100.77h4.243v-2.684c0-.156-.066-.234-.196-.234h-4.046v-6.655l-.196-.584zM6.44 93.221H7.8c4.723 0 7.085 1.737 7.085 5.215 0 1.608-.508 2.893-1.52 3.853-1.141 1.09-2.905 1.635-5.29 1.635H6.438V93.22zm40.626 4.627c-1.66 0-3.088.441-4.28 1.324-1.273.96-1.907 2.232-1.907 3.814 0 1.608.584 2.905 1.752 3.89.57.468 1.277.91 2.121 1.325a64.24 64.24 0 0 0 3.035 1.38c1.18.507 1.77 1.239 1.77 2.198 0 .752-.298 1.324-.895 1.713a3.13 3.13 0 0 1-.855.352c-.312.078-.72.117-1.225.117-.506 0-1.058-.131-1.654-.39l-.934-2.257-4.01 1.518c.39 1.141 1.219 2.103 2.49 2.88 1.375.806 2.79 1.208 4.243 1.208 1.738 0 3.219-.419 4.437-1.248A5.214 5.214 0 0 0 52.672 114c.39-.674.584-1.538.584-2.59a4.625 4.625 0 0 0-.973-2.86c-.336-.467-.81-.907-1.42-1.321-.61-.416-1.484-.858-2.627-1.325-1.141-.467-2.036-.913-2.685-1.34-.65-.427-.973-.98-.973-1.654 0-.675.247-1.186.74-1.537a2.765 2.765 0 0 1 1.635-.525c.597 0 1.155.129 1.674.388l.894 1.985 3.97-1.518a6.95 6.95 0 0 0-2.296-2.607 7.232 7.232 0 0 0-4.127-1.244l-.002-.004zm-17.627.004c-2.335 0-4.253.804-5.76 2.412-.778.804-1.39 1.79-1.83 2.957-.441 1.167-.662 2.503-.662 4.01 0 2.957.78 5.304 2.336 7.042 1.53 1.764 3.504 2.647 5.916 2.647 2.308 0 4.229-.83 5.76-2.49 1.662-1.739 2.49-4.112 2.49-7.121 0-2.88-.777-5.177-2.334-6.89-1.556-1.711-3.53-2.567-5.916-2.567zm0 2.918c1.402 0 2.51.606 3.327 1.812.818 1.207 1.226 2.885 1.226 5.04 0 2.153-.429 3.755-1.285 4.804-.856 1.051-1.957 1.576-3.307 1.576-1.348 0-2.44-.537-3.27-1.615-.83-1.076-1.245-2.659-1.245-4.748 0-2.09.376-3.743 1.129-4.961.804-1.272 1.947-1.908 3.425-1.908z"/>
</svg>
```

Reply:
thx @Snailedlt, i updated the svg files